websites/V/VGMDB/dist/metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.6",
+  "author": {
+    "name": "Kyrie",
+    "id": "368399721494216706"
+  },
+  "service": "VGMDB",
+  "description": {
+    "en": "VGMdb is a community project dedicated to cataloging the music of visual arts and video games.",
+    "vi_VN": "VGMdb là dự án cộng đồng dành riêng cho việc lưu trữ âm nhạc và tranh ảnh của trò chơi điện tử."
```

Comment:
db in caps in service but not when mentioned in description... which is correct?

Reply:
The one without the caps is correct I think, but I accidentally pushed with the folder name with all caps and have no idea how to change it.
I can make a new PR if you want

Reply:
just rename the folder in vscode and push it should just remove the old and add the new

Reply:
I tried renaming but it didn't count as a change for some reason, will look into it ASAP

Reply:
Done